docs/README-develop-beaker-scripts.md

```diff
@@ -135,6 +199,18 @@ HOSTS:
           user: devops
           password: devopspassword
 
+    xr-agent:
+        roles:
+            - agent
+        platform: cisco-7-x86_64
```

Comment:
The example hosts.cfg will need to be updated if/when puppetlabs/beaker#1066 is merged - just an FYI, not related directly to this PR.